tools/RAiDER/models/weatherModel.py

```diff
@@ -27,6 +27,11 @@ class WeatherModel(ABC):
     '''
 
     def __init__(self):
+        # without a better way to solve the problem, suggest to add this back ; otherwise missing a lot of logging info needed for all other models e.g. GMAO (except those call cdsapi directly like ERA-5)  -- Yang (update: Dec 15 2020)
```

Comment:
I recommend not to include this here since it is not related to the netcdf. We can include it back in its own PR.